nixos/modules/misc/mindustry.nix

```diff
@@ -0,0 +1,41 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.services.mindustry-server;
+in {
+  options = {
+    services.mindustry-server = {
+      enable = mkEnableOption "Mindustry server";
+
+      package = mkPackageOption pkgs "mindustry-server" {};
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Open ports in the firewall for the mindustry server";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.mindustry = {
+      enable = true;
```

Comment:
Services are enabled by default. This should stay unset, so that users can disable them if needed.
```suggestion
```